### DISCUSSION TITLE
The repository I stars does not push updates to my homepage

### DISCUSSION THREAD

**Discussion by KKISO574** [community/community#146712] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

This problem has been going on for a month. Many of the repositories I follow have not been notified of updates, but when I find them, I find that they have updated content, but I can't receive it.

**Comment by IndianPhotoAcademy** [comment]
If the repository you starred isn't updating on your homepage, it could be due to caching or delayed sync between GitHub's backend and the interface. Try refreshing your browser or clearing its cache. Also, ensure the repository’s visibility hasn’t changed, and confirm GitHub's servers are functioning without outages.

**Comment by Lut-Lat-Aung** [comment]
Hi, @KKISO574 
I think it is because you just star the repository or do you also click 'watch' ?

Starring a repository doesn’t subscribe you to its updates. To see recent changes on your homepage, you need to “Watch” the repository instead of (or in addition to) just starring it.

Make sure your GitHub notifications are enabled. Visit Settings > Notifications to confirm your preferences.

I hope you find my answers helpful.
